src/components/Error.tsx

```diff
@@ -1,16 +1,30 @@
-import { FunctionComponent } from 'react'
+import { FunctionComponent, useEffect, useState } from 'react'
 import { Alert } from 'react-bootstrap'
 import { ErrorType } from '../context/error'
 
 interface ErrorProps extends ErrorType {
   onClose: () => void
 }
 
-const Error: FunctionComponent<ErrorProps> = ({ title, body, onClose }) => (
-  <Alert variant='danger' onClose={onClose} dismissible>
-    <Alert.Heading>{title ?? 'Error'}</Alert.Heading>
-    <p>{body}</p>
-  </Alert>
-)
+const Error: FunctionComponent<ErrorProps> = ({ title, body, variant, delay = 3000, onClose }) => {
```

Comment:
If we keep this an Error at least variant should default to danger in this line just like delay has a default too.
We can make it consistent for title too to have a default here and skip the ternary shorthand below